BD-3885 Export logs EA (#8892)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/exports_log.md

---
nav_title: Exports Log

> article_title: Exports Log

Reviewer: ```suggestion
article_title: Exports Log
page_order: 15
```
Not sure if this is the page order you want, but adding this to the YAML!

page_type: reference
description: "This page covers the exports log, which lets you view the status of export jobs and cancel ongoing exports."
---

# Exports log

> Use the **Exports Log** page to view the status of export jobs and cancel ongoing exports directly from the Braze platform. <br><br> Currently, the exports log only supports segment exports. For information about the segment export tool, refer to [Exporting segment data to CSV]({{site.baseurl}}/user_guide/data/export_braze_data/segment_data_to_csv/).

{% alert important %}
The exports log is in early access. If you're interested in participating in the early access, reach out to your customer success manager.
{% endalert %}

> You can find the exports log by going to **Settings** > **Exports Log**. Here, you can view previous exports, the name of exported segments, the status for each export, the source of each export, and when each export started and finished. 

Reviewer: ```suggestion
You can find the exports log by going to **Settings** > **Exports Log**. Here, you can view previous exports, the name of exported segments, the status of each export, the source of each export, and when each export started and finished. 
```

![The export logs with a list of completed exports.][1]

## Cancelling a pending export

> You can cancel pending exports directly from the **Exports Log** page by selecting the vertical three dot icon and then selecting **Cancel Export**, or selecting the **Export ID** and then selecting **Cancel Export** on the export's page.

Reviewer: ```suggestion
You can cancel pending exports directly from the **Exports Log** page by selecting the <i class="fas fa-ellipsis-vertical"></i> menu and then selecting **Cancel Export**, or selecting the **Export ID** and then selecting **Cancel Export** on the export's page.
```
